examples/lightgbm_tuner_simple.py

```diff
@@ -0,0 +1,53 @@
+"""
+Optuna example that optimizes a classifier configuration for cancer dataset using LightGBM tuner.
+
+In this example, we optimize the validation accuracy of cancer detection.
+We optimize both the choice of booster model and their hyperparameters.
+
+You can execute this code directly.
+    $ python lightgbm_tuner_simple.py
+
+"""
+
+import numpy as np
+import sklearn.datasets
+from sklearn.metrics import accuracy_score
+from sklearn.model_selection import train_test_split
+
+import lightgbm as lgb_original
```

Comment:
In this example, `import lightgbm` can be omitted.
`optuna.integraiton.lightgbm.Dataset` can be used instead of `lightgbm.Dataset`.

```diff
diff --git a/examples/lightgbm_tuner_simple.py b/examples/lightgbm_tuner_simple.py                                                                                                                                               
index 5d325e19..1bd78e1b 100644
--- a/examples/lightgbm_tuner_simple.py
+++ b/examples/lightgbm_tuner_simple.py
@@ -14,15 +14,14 @@ import sklearn.datasets
 from sklearn.metrics import accuracy_score
 from sklearn.model_selection import train_test_split
 
-import lightgbm as lgb_original
 import optuna.integration.lightgbm as lgb
 
 
 if __name__ == '__main__':
     data, target = sklearn.datasets.load_breast_cancer(return_X_y=True)
     train_x, val_x, train_y, val_y = train_test_split(data, target, test_size=0.25)
-    dtrain = lgb_original.Dataset(train_x, label=train_y)
-    dval = lgb_original.Dataset(val_x, label=val_y)
+    dtrain = lgb.Dataset(train_x, label=train_y)
+    dval = lgb.Dataset(val_x, label=val_y)
 
     params = {
         'objective': 'binary',

Reply:
Good catch!
I have updated the code as you suggested.